packages/cli/src/utils/governance.ts

```diff
@@ -0,0 +1,25 @@
+import { CeloTxPending } from '@celo/connect'
+import { ContractKit } from '@celo/contractkit'
+
+export async function checkProposal(
+  proposal: Array<Pick<CeloTxPending, 'to' | 'input' | 'value'>>,
+  kit: ContractKit
+) {
+  const governance = await kit.contracts.getGovernance()
+
+  for (const tx of proposal) {
+    console.log(tx)
+    if (!tx.to) {
+      continue
+    }
+
+    console.log(
+      await kit.web3.eth.call({
+        to: tx.to,
+        from: governance.address,
+        value: tx.value,
+        data: tx.input,
+      })
+    )
+  }
```

Comment:
does it make sense to always print the output here? across `propose`, `build`, etc.?
maybe we should only print errors?

Reply:
also maybe we should `try catch` all of the calls separately so we see an error with transaction n+1 even transaction n also fails